### PR TITLE
Pull request sprint 1

### PR DESCRIPTION
**Pull request by BugelNiels**

### CODE REVIEW

File: .idea/libraries/Java_EE_6_Java_EE_6.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<component name="libraryTable">
```
Comment (adeng322) [inline .idea/libraries/Java_EE_6_Java_EE_6.xml]: please dont submit any .idea file

File: OrientDBVisualizationTool/.settings/org.eclipse.core.resources.prefs [hunk 2]
```diff
@@ -0,0 +1,4 @@
+eclipse.preferences.version=1
```
Comment (adeng322) [inline OrientDBVisualizationTool/.settings/org.eclipse.core.resources.prefs]: please ignore all these files in your submission

File: OrientDBVisualizationTool/src/main/java/com/mycompany/orientdbvisualizationtool/database/Test.java [hunk 3]
```diff
@@ -0,0 +1,99 @@
+/*
```
Comment (adeng322) [inline OrientDBVisualizationTool/src/main/java/com/mycompany/orientdbvisualizationtool/database/Test.java]: if entire file has been commented out, why you included it in your submission?

File: OrientDBVisualizationTool/src/main/java/com/mycompany/orientdbvisualizationtool/model/Orginazation.java [hunk 4]
```diff
@@ -0,0 +1,14 @@
+/*
```
Comment (adeng322) [inline OrientDBVisualizationTool/src/main/java/com/mycompany/orientdbvisualizationtool/model/Orginazation.java]: what does this class do?

Comment (BugelNiels) [reply OrientDBVisualizationTool/src/main/java/com/mycompany/orientdbvisualizationtool/model/Orginazation.java]: As of right now, it is empty(so only a skeleton file). However, this class will be used to keep data of a specific organization. We added some more content to this file

**Pull request merged**